api/src/main/java/org/apache/xtable/model/TableSyncMetadata.java

```diff
@@ -41,7 +41,7 @@ public class TableSyncMetadata {
    * Property name for the lastInstantSynced field from SyncResult, used for persisting
    * lastInstantSynced in the table metadata/properties
    */
-  public static final String ONETABLE_LAST_INSTANT_SYNCED_PROP = "ONETABLE_LAST_INSTANT_SYNCED";
+  public static final String XTABLE_LAST_INSTANT_SYNCED_PROP = "XTABLE_LAST_INSTANT_SYNCED";
```

Comment:
@ashvina this is probably the most controversial part of the PR so I would like your thoughts on doing this backwards incompatible change

Reply:
another option is to have a legacy value we parse if this value is not present

Reply:
I've added another ticket to change how we are doing this metadata tracking. I think we should just go for the breaking change now before we do our first release: https://github.com/apache/incubator-xtable/issues/412